lib/order.rb

```diff
@@ -0,0 +1,51 @@
+class Order
+
+  def initialize
+    @dish_list_with_prices = {'Egg Fried Rice' => '3', 'Prawn Crackers' => '2', 'Sweet and Sour Chicken' => '5'}
```

Comment:
Please could you try to use the `Menu` class here to remove the repetition of the dish list?